### PR TITLE
[NRLF-644] errors in swagger consumer

### PR DESCRIPTION
**Pull request by jameslinnell**

### CODE REVIEW

**Review by nomad3k — CHANGES_REQUESTED** [review]
The `/_count` endpoint is missing the Errors, because this is the only endpoint that resides in it's own swagger file.

**Review by nomad3k — APPROVED** [review]

File: api/consumer/swagger.yaml [hunk 1]
```diff
@@ -606,6 +697,34 @@ paths:
         - $ref: "#/components/parameters/requestId"
         - $ref: "#/components/parameters/correlationId"
       responses:
+        "4xx":
```
Comment (axelkrastek1-nhs) [inline api/consumer/swagger.yaml:700]: Should this be uppercase as well?

File: api/consumer/record-locator/consumer.yaml [hunk 2]
```diff
@@ -567,6 +658,34 @@ paths:
         - $ref: "#/components/parameters/requestId"
         - $ref: "#/components/parameters/correlationId"
       responses:
+        "4xx":
```
Comment (axelkrastek1-nhs) [inline api/consumer/record-locator/consumer.yaml:661]: Should this be uppercase as well?

File: swagger/consumer-static/count.yaml [hunk 3]
```diff
@@ -12,6 +12,34 @@ paths:
         - $ref: "#/components/parameters/requestId"
         - $ref: "#/components/parameters/correlationId"
       responses:
+        "4xx":
```
Comment (axelkrastek1-nhs) [inline swagger/consumer-static/count.yaml:15]: Should this be uppercase as well?

**Pull request merged**